native/dlabad.go

```diff
@@ -0,0 +1,11 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+func (impl Implementation) Dlabad(small, large float64) {
```

Comment:
I'll leave it up to you as to whether it's kept, but it seems to me that if it is kept, either you accept pointers (something we avoid) or return the values unaltered.